ocaml/parsing/jane_syntax_parsing.mli

```diff
@@ -121,10 +147,16 @@ module Embedded_name : sig
   *)
   type components = ( :: ) of string * string list
 
-  (** The trailing components together with the erasability. The erasability is
-      the second component of the literal name that appears in the parsetree.
+  type t
```

Comment:
Might making it `private` be better?  Then we don't need separate accessors and such.

Reply:
Making it private stands at odds with not exposing `erasability` -- sadly we can't expose just `components` for pattern matching.

Reply:
(and fwiw I like not needing to expose `Erasability.t` in the mli -- I get that we could do something like only expose `module Erasability : sig type t end` in the mli, but this seems weirder than just having an accessor.)

Reply:
I missed that!  Yeah, in that case, sold -- there's room to expose the strings "erasable" and "non_erasable" somewhere since code needs them, but this seems good.